python/plugins/db_manager/db_plugins/plugin.py

```diff
@@ -1095,13 +1095,31 @@ def definition(self):
             txt += u" DEFAULT %s" % self.default2String()
         return txt
 
+    def getComment(self):
+        # Function returning the comment of a field if exists
+        tab = self.table()
+            #SQL Query checking if a comment exists for the field
+        sql_cpt = u"Select count(*) from pg_description pd, pg_class pc, pg_attribute pa where relname = '%s' and attname = '%s' and pa.attrelid = pc.oid and pd.objoid = pc.oid and pd.objsubid = pa.attnum" % (tab.name, self.name)
+            #SQL Query that return the comment of the field
+        sql = u"Select pd.description from pg_description pd, pg_class pc, pg_attribute pa where relname = '%s' and attname = '%s' and pa.attrelid = pc.oid and pd.objoid = pc.oid and pd.objsubid = pa.attnum" % (tab.name, self.name)
+        c = tab.database().connector._execute(None, sql_cpt)# Execute Check query
+        res = tab.database().connector._fetchone(c)[0] #Store result
+        if res == 1: 
+            # When a comment exists
```

Comment:
```suggestion            
```

Reply:
Is that a wrong statement?
A clarification of the meaning of `res ==1` seems useful